testsuite/features/secondary/srv_retracted_patches.feature

```diff
@@ -0,0 +1,115 @@
+    Then I should see "sle_minion" hostname
+   
+  Scenario: Target systems for retracted packages should be empty
+    Given I am authorized as "admin" with password "admin"
+    When I follow the left menu "Software > Channel List > All"
+    And I follow "Test-Channel-x86_64"
+    And I follow "Packages" in the content area
+    And I follow "rute-dummy-2.1-1.1.x86_64"
+    And I follow "Target Systems"
+    Then I should not see "sle_minion" hostname
+
+  Scenario: Retracted packages in the patch detail
+    Given I am authorized as "admin" with password "admin"
+    When I follow the left menu "Patches > Patch List > All"
+    And I follow "rute-dummy-0815"
+    Then I should see a "Status: Retracted" text
+    When I go back
+    And I follow "rute-dummy-0816"
+    Then I should see a "Status: Stable" text
+    When I go back
+    And I follow "rute-dummy-0817"
+    Then I should see a "Status: Retracted" text
+
+  Scenario: Retracted packages in the patches list
+    Given I am authorized as "admin" with password "admin"
+    When I follow the left menu "Patches > Patch List > All"
+    Then the table row for "rute-dummy-0815" should contain "errata-retracted" icon
+    Then the table row for "rute-dummy-0816" should not contain "errata-retracted" icon
+    Then the table row for "rute-dummy-0817" should contain "errata-retracted" icon
+
+  Scenario: Retracted patches in the channel patches list
+    Given I am authorized as "admin" with password "admin"
+    When I follow the left menu "Software > Channel List > All"
+    And I follow "Test-Channel-x86_64"
+    And I follow "Patches" in the content area
+    Then the table row for "rute-dummy-0815" should contain "errata-retracted" icon
+    Then the table row for "rute-dummy-0816" should not contain "errata-retracted" icon
+    Then the table row for "rute-dummy-0817" should contain "errata-retracted" icon
+ 
+  Scenario: Retracted packages in the channel packages list
+    Given I am authorized as "admin" with password "admin"
+    When I follow the left menu "Software > Channel List > All"
+    And I follow "Test-Channel-x86_64"
+    And I follow "Packages" in the content area
+    Then the table row for "rute-dummy-2.0-1.1.x86_64" should contain "errata-retracted" icon
+    Then the table row for "rute-dummy-2.0-1.2.x86_64" should not contain "errata-retracted" icon
+    Then the table row for "rute-dummy-2.1-1.1.x86_64" should contain "errata-retracted" icon
+
+  Scenario: SSM: Retracted package should not be available for installation
+    Given I am authorized as "admin" with password "admin"
+    When I am on the System Overview page
+    And I follow "Clear"
+    And I check the "sle_minion" client 
+    And I am on System Set Manager Overview
+    And I follow "Packages" in the content area
+    And I follow "Install"
+    And I follow "Test-Channel-x86_64"
+    Then I should see a "rute-dummy-2.0-1.2" text
+    Then I should not see a "rute-dummy-2.1-1.1" text
+
```

Comment:
no final empty line

Reply:
Yes, it is idempotent. The only modification done is installing the package on a system in some scenarios. These scenarios clean the package at the end (`I remove package "rute-dummy"...`).

Reply:
it would indeed be cleaner to install the package in an initial (separated) scenario, and to remove it in a final (separated) cleanup scenario (with prefix `Cleanup: `), for the reason you explain.

For me, it's good enough like you currently clean up. But feel free to be more radical if you prefer.

Reply:
I could group the scenarios that install the package together. But: there are 3 scenarios that install the package, but only 2 of them install the same version. So I think we can just leave this as is.